carbon-design-system/carbon
# feat(Accordion): adds flush alignment

Closes #8881


#### Changelog

- adds conditional className and styles based on new prop `isFlush`
- only works if `align = "end"`
- adds tests 

#### Testing / Reviewing
- go to Accordion > Playground story + Accordion > Skeleton story
- toggle `isFlush` prop
- Accordion text should be aligned to the left
- If the prop `align` = `start`, `flush` should not work
- Check that all tests are passing
